package.json

```diff
@@ -145,6 +149,12 @@
         }
       ]
     },
+    "viewsWelcome": [
+      {
+        "view": "idfSearchResults",
+        "contents": "On any opened file editor, select some text and right click and select **ESP-IDF: Search in documentation** to get matching results here.\n\nResults are based on your current vscode language, `idf.espIdfPath` version (latest otherwise) and `idf.adapterTargetName`."
```

Comment:
I think markdown is not rendered here properly (at least in windows), could you check again, if its supported else we can remove the syntax!

![Screen Shot 2020-11-30 at 2 40 49 PM](https://user-images.githubusercontent.com/25585652/100590185-0b7f7200-331a-11eb-9a79-09a28c156acb.png)